python/ray/tests/test_cli.py

```diff
@@ -492,11 +496,14 @@ def output_ready():
 
     result_env_arg = runner.invoke(scripts.status, ["--address", address])
     _check_output_via_pattern("test_ray_status.txt", result_env_arg)
-    ray.shutdown()
 
 
-@pytest.mark.xfail(cluster_not_supported, reason="cluster not supported")
-def test_ray_status_multinode():
+@pytest.mark.xfail(
+    cluster_not_supported, reason="cluster not supported on Windows")
+def test_ray_status_multinode(shutdown_only):
```

Comment:
Shouldn't we shut down the cluster as well? Should we use `ray_start_cluster` instead?


Btw, sleep(2) seems dangerous and not a good practice. How about we skip this test for now and fix it as part of CI tests fixing?